packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Shared/WebHostServerFixture.cs

```diff
@@ -20,6 +20,8 @@ public async Task InitializeAsync()
         ServerAddress = new Uri(addressFeature!.Addresses.First());
     }
 
+    protected abstract Startup StartupFactory(WebHostBuilderContext context);
```

Comment:
Even though we don't use the `context` parameter, it's what the `UseStartup()` call below wants.